tutorials/eth-api/randomness-lottery.md

```diff
@@ -445,7 +445,7 @@ async function main() {
   console.log('Deploying Lottery...');
 
   // 4. Waiting for the deployment to resolve
-  await lottery.deployed();
+  await lottery.waitForDeployment();
```

Comment:
closest I could get to the line that needs to be updated.

Line 451 should use `lottery.target` as `address` is no longer valid and as it is right now it returns `undefined`